ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -316,3 +316,8 @@ def getCatalogPath( catalogName ):
   """  Return the configuration path of the description for a a given catalog
   """
   return '/Resources/FileCatalogs/%s' % catalogName
+
+def getBackendConfig(backendID):
```

Comment:
The attribute is more backendID as we said, because it can be 'stdout' as 'f051'.
I added a comment to explain the choice.